client/landing/gutenboarding/available-designs.ts

```diff
@@ -34,9 +35,17 @@ export const getDesignImageUrl = ( design: Design ) => {
 		{
 			font_headings: design.fonts.headings,
 			font_base: design.fonts.base,
+			viewport_height: 700,
 		}
 	);
-	return mshotsUrl + encodeURIComponent( previewUrl );
+	const mshotsRequest = addQueryArgs( mshotsUrl + encodeURIComponent( previewUrl ), {
+		vpw: 1200,
+		vph: 3072,
+		w: 700,
```

Comment:
Might be better to have `700` in constant so that nobody misses updating it between `previewUrl` and `mshotsRequest`?